src/main/resources/org/biouno/unochoice/stapler/unochoice/unochoice.js

```diff
@@ -364,44 +230,23 @@ var UnoChoice = UnoChoice || ($ => {
                         for (let i = 0; i < newValues.length; i++) {
                             let entry = newValues[i];
                             let key = newKeys[i];
-                            // <TR>
-                            let tr = document.createElement('div');
                             let idValue = `ecp_${_self.getRandomName()}_${i}`;
                             idValue = idValue.replace(' ', '_');
-                            tr.setAttribute('id', idValue);
-                            tr.setAttribute('style', 'white-space:nowrap');
-                            tr.setAttribute('class', 'tr');
-                            // <TD>
-                            let td = document.createElement('div');
                             // <INPUT>
-                            let input = document.createElement('input');
-                            // <LABEL>
-                            let label = document.createElement('label');
-                            if (selectedElements.indexOf(i) >= 0) {
-                                input.setAttribute('checked', 'checked');
-                            }
-                            if (disabledElements.indexOf(i) >= 0) {
-                                input.setAttribute('disabled', 'disabled');
-                            }
-                            input.setAttribute('json', key);
-                            input.setAttribute('name', 'value');
-                            input.setAttribute("value", key);
-                            input.setAttribute("class", " ");
-                            input.setAttribute("type", "checkbox");
-                            label.className = "attach-previous";
+                            let input = makeCheckbox(key, selectedElements.indexOf(i) >= 0, disabledElements.indexOf(i) >= 0);
```

Comment:
This is an example of a helper method I've introduced where we've had multiple places where we were creating checkboxes.

Reply:
Brilliant simplification! :clap: